shared/src/settings/dynamicSettings.ts

```diff
@@ -17,7 +17,17 @@ export const dynamicSettings: DynamicSettings = {
   },
   header: {
     showFeedback: true,
-    showHelp: true
+    showHelp: true,
+    topBar: {
+      dark: {
+        overImgBgColor: "transparent"
```

Comment:
We probably want to set `bgColor: 'oklch(var(--b3))'` as the default as well.

Btw, to use a semi-transparent version thereof for overImgBgColor, we can use `'oklch(from oklch(var(--b3)) l c h / 0.5)'`, but `transparent` is a better default.